### PR TITLE
OSDOCS-4650 adding relnotes template adoc to MicroShift 4.12

### PR DESCRIPTION
**Pull request by ShaunaDiaz**
[OSDOCS-4650](https://issues.redhat.com//browse/OSDOCS-4650) adding relnotes template adoc to MicroShift 4.12

Applies to: 4.12 only

Issue:
https://issues.redhat.com/browse/OSDOCS-4650

Link to docs preview:
https://53503--docspreview.netlify.app/microshift/latest/microshift_release_notes/microshift-4-12-release-notes.html

Additional information: This is a placeholder for later additions. Thank you!

### CODE REVIEW

File: microshift_release_notes/microshift-4-12-release-notes.adoc [hunk 1]
```diff
@@ -1,13 +1,23 @@
...
 include::_attributes/attributes-microshift.adoc[]
 :context: release-notes
 
-Do not add or edit release notes here. Edit release notes directly in the branch
-that they are relevant for.
+toc::[]
 
-Release note changes should be added/edited in their own PR.
+// Update All for MicroShift; text is FPO (for placement only)
+Red Hat {product-title} provides developers and IT organizations with an OpenShift-like experience for small form factor and edge computing, delivered as an application that customers can deploy on top of their managed RHEL devices at the edge. {product-title} is designed to make control plane restarts economical and be lifecycle-managed as a single unit by the operating system. Updates, roll-backs, and config changes become a matter of staging another version in parallel and then - without relying on a network - flipping to/from that version and restarting.
```
Comment (tmalove) [inline microshift_release_notes/microshift-4-12-release-notes.adoc:10]:
```suggestion
Red Hat {product-title} provides developers and IT organizations with an OpenShift-like experience for small form factor and edge computing, delivered as an application that customers can deploy on top of their managed RHEL devices at the edge. {product-title} is designed to make control plane restarts economical and be lifecycle-managed as a single unit by the operating system. Updates, roll-backs, and configuration changes become a matter of staging another version in parallel and then - without relying on a network - flipping to or from that version and restarting.
```
to/from: see https://www.ibm.com/docs/en/ibm-style?topic=punctuation-slashes
config: see https://www.ibm.com/docs/en/ibm-style?topic=grammar-abbreviations

I am not sure about, and cannot find any guidance on using 'OpenShift-like'.

Comment (tmalove) [reply microshift_release_notes/microshift-4-12-release-notes.adoc:10]: @ShaunaDiaz  for the preview, can you generate one without the slashes so we can see what the rendering is (and then add them back per your needs)? Thanks!

Comment (tmalove) [reply microshift_release_notes/microshift-4-12-release-notes.adoc:10]: /remove-label peer-review-in-progress
/label peer-review-done

**Comment by mjpytlak** [comment]
@ShaunaDiaz To confirm, this is placeholder work and will not be published in 4.12? I ask because the PR does not have QE approval.

**Comment by ShaunaDiaz** [comment]
@mjpytlak Correct, I am just building the place for relnotes to go.

**Pull request merged**